src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -482,6 +482,17 @@ bool fx_muxer_t::resolve_hostpolicy_dir(host_mode_t mode,
 
     // If it still couldn't be found, somebody upstack messed up. Flag an error for the "expected" location.
     trace::error(_X("A fatal error was encountered. The library '%s' required to execute the application was not found in '%s'."), LIBHOSTPOLICY_NAME, expected.c_str());
+    if (mode == host_mode_t::muxer && !config.get_portable())
+    {
+        if (!pal::file_exists(config.get_path()))
+        {
+            trace::error(_X("Running as a standalone app because there is no '%s' file. If this should be a portable app instead, add that file specifying the appropriate framework."), config.get_path().c_str());
```

Comment:
(minor) Instead of saying `Running as a standalone app`, would it be better to say `Attempted to run as a standalone app`?  Since we didn't really run anything.

Reply:
Hmm. I changed it to "Failed to run as a standalone app"

Reply:
Sounds good to me.  Thanks.